app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+import os
+import re
+
+
+def move_file(command: str) -> None:
+    try:
+        command, source_path, destination_path = command.split()
+    except ValueError:
+        print("Wrong command format")
+    else:
+        if command == "mv" and source_path != destination_path:
+            if not os.path.isfile(source_path):
+                raise ValueError(f"File '{source_path}' doesn't exist")
+            source_path = os.path.join(*re.split(r"[\\/]", source_path))
+            destination_path = os.path.join(
+                *re.split(r"[\\/]", destination_path)
+            )
+            if (
+                    source_path.endswith(os.sep)
+                    or destination_path.endswith(os.sep)
+            ):
+                return
+
+            if os.sep in destination_path:
```

Comment:
using `os.path.isdir(destination_path)` would be a preferable way to check if path is a dir or not